ext/hal/st/lib/sensor/vl53l0x/api/platform/inc/vl53l0x_platform.h

```diff
@@ -62,6 +62,7 @@ typedef struct {
     uint8_t   I2cDevAddr;                /*!< i2c device address user specific field */
     uint8_t   comms_type;                /*!< Type of comms : VL53L0X_COMMS_I2C or VL53L0X_COMMS_SPI */
     uint16_t  comms_speed_khz;           /*!< Comms speed [kHz] : typically 400kHz for I2C           */
+    struct device *i2c;

```

Comment:
this doesn't seem like it should be required or how we support things.

Reply:
Not sure I well understood your remark here. This i2c pointer is used here to store the pointer on i2c device. This is obtained by a call to device_get_binding(CONFIG_VL53L0X_I2C_MASTER_DEV_NAME) in the driver and then, I use it on the lib part to deal with i2c.
In my opinion, we need it. Perhaps you prefer to handle this in an other way ?